lib/actions/jlinkTargetActions.js

```diff
@@ -592,7 +574,39 @@ const recoverOneCore =
                     }
                 );
             });
+        };
+
+        if (family === 'nRF52' || family === 'nRF53') {
+            let recoverFile;
+            recoverFile =
+                family === 'nRF52'
+                    ? recoverHex(getDeviceType(serialNumberWithCore))
+                    : recoverFile;
+            recoverFile =
+                family === 'nRF53'
+                    ? recoverHex53[core.coreNumber]
+                    : recoverFile;
```

Comment:
So if this if evaluates to true, we already know that the `family` is either `nRF52` or `nRF53`, so we can simplify this:

```suggestion
        if (family === 'nRF52' || family === 'nRF53') {
            const recoverFile = family === 'nRF52' ? recoverHex(getDeviceType(serialNumberWithCore)) : recoverHex53[core.coreNumber]

```